onnxruntime/core/providers/dml/DmlExecutionProvider/src/Operators/DmlOperatorQAttention.cpp

```diff
@@ -41,7 +41,7 @@ class DmlOperatorQAttention : public DmlOperator
     :   DmlOperator(kernelCreationContext)
     {
 
-        enum DmlInputIndex : uint32_t
+        enum inputIndex : uint32_t
```

Comment:
Why is this being renamed to a less clear name and using a casing that is inconsistent with the other two enums right below?